elastic/integrations
# [GCP] Add various dashboards

## What does this PR do?



Some dashboards have not been migrated during the migration from `gcp` beat module to the integration package.

This PR adds the missing ones, updating them to the latest dashboard development guidelines.

Updates:
- GKE overview dashboard
- PubSub metrics overview dashboard
- Audit events overview dashboard
- Storage metrics overview dashboard

GCP package is bumped to 2.17.0.

## Checklist


## Author's Checklist



## How to test this PR locally



- Start Elastic cluster with `elastic-package stack up --version 8.6.1`
- Collect data for GKE, PubSub and Storage metrics by adding a policy with appropriate credentials
- Collect data for Audit Logs; default policy information should be enough, as the required collection infrastructure is already in place
- verify dashboard show data and filters work as expected

## Related issues


-

## Screenshots



### GCP Audit logs
![Screenshot 2023-02-13 at 17-59-59 Elastic](https://user-images.githubusercontent.com/526307/218523887-5eb2ff09-ff70-4332-b9ea-5b0fe9eda75c.png)

### GCP GKE
![Screenshot 2023-02-13 at 18-04-30 Elastic](https://user-images.githubusercontent.com/526307/218523911-9b553f4d-8282-42ea-a738-cef73a0e20a9.png)

### GCP Storage
![Screenshot 2023-02-13 at 18-04-35 Dashboards - Elastic](https://user-images.githubusercontent.com/526307/218523920-c9d9bdf2-28ed-43bb-93ec-425bbd6f26d3.png)

### GCP PubSub
![Screenshot 2023-02-13 at 18-04-39 Elastic](https://user-images.githubusercontent.com/526307/218523944-1da17606-4c09-48e9-959c-196354e6ce87.png)
